mmv1/products/cloudbuild/Trigger.yaml

```diff
@@ -42,6 +42,11 @@ examples:
   - !ruby/object:Provider::Terraform::Examples
     name: "cloudbuild_trigger_service_account"
     primary_resource_id: "service-account-trigger"
+    vars:
+      sa_name: "my-service-account"
+    test_vars_overrides:
+      sa_name: "fmt.Sprintf(\"tf-test-cloud-sa-%s\", context[\"random_suffix\"\
```

Comment:
I removed the test_vars_overrides but ran into a char limit for the Service Account name (30 char limit) so I added it back and tailored the name to be less than 30 char. Since these examples are also used for the documentation I think this would make sense

[https://ff3ece52349aad93016ad0c53f4fb026dfde1721c86b911dff8926e-apidata.googleusercontent.com/download/storage/v1/b/ci-vcr-logs/o/beta%2Frefs%2Fheads%2Fauto-pr-7543%2Fartifacts%2F0092990f-d3f7-4be2-91aa-5971a0031a79%2Fbuild-log%2Frecording_build%2FTestAccCloudBuildTrigger_cloudbuildTriggerServiceAccountExample_recording_test.log?jk=ARTXCFFgTwso7bnwOPLV1C4JbmoGFTnngpv52yLq4BIkg4VHBwpYn_BPFgfqZey5q8GeLuPso7Itm5yGO0P1RCEJNcN0uD8yQ9TXMdvRcPol63F9oq0nHKUiY910NXkMglNjnVAHb0gGB9RTajTnnZ45VXCjIajPhprlJX0LixvcdMU2M11tcCzUWmBS7vLxWujZGtFQbWgFEH1vn5Z3WvDhEPoeliCK4PwEP2SFYyMyTGIsiItba5KtlEQ0JI4i5jkdkncmKAuT7PuQxRXSeEBVZJu2NZn3wQv86ppLB7M68ld_DF_Jg-Vfy8fALIKXytrRgUBHXfKupuWVdH7PTHdqZkt5lgZOd9mN_MojS-6mvWKOhS75kjKYNwmtlZUHvpm2h20YrZ2CnFEHOnlfCKi__yzZNInVw8hL1ZdcNIkckafmnOTsi3OSLK2yRhvfYD6MgZZL5DYnuytHf76TevOOf0QPe_LF-afBnEFIqmdp0MnbtvO2CPD2L9twDmFfnFJItVIraEcqj5Ej9iYL-L0n-DJMq-bzgocpOfQNfNu6XwoFRdstXYP0XG271rIFegvf-JFAjIis2n5rCbXbmq01J6aNde3MeqMFk3Kf5lYu-iJJZAajrzTP05EBM_T0OfG72rDyIdIKyiOs7CgvEHV73bgmMvyFZ2iYkD5FN60ETumIWNehSBkKDQogkFnnk_SPNDrTO9eefCt9gsKQH7JaDazNZZ5LEz2orQTLKG8aEflqwqh_dfbtDoNYqjnDp5qHL6B-Bv0hysRc6-OB5BpSHIL3Za8NxPhEVaNOxoTzhFRmqau07V6zt1zBxhEmYkKrQBKYbQF2ZgLCjBtszRynRPrSP7A8y2zanYaIqecH9O0BiR_uLGA-KXWGM_jZ4IOTnyljO9iS7YyGtlQgVkVzj-DDZO7KfcWnHk3DTvOUT6eQX4i6SZweWJiUMgEusvVMsHfNWvt5yfvE6-BcfIqlWwfxgOm-E4bwALSE7SgzMWv5-v372iSQ6XtCcwIFffhilws0xeYZjdGJmVSM_gBZb1h5ZZq20vB3MYUvWpZVm413DgeOe4W-eqYajeA-dMynPBgZ9Z5c4b7ug59KWJdt4beaao2ISW1jjhS6s9mpjjuaZiHu4F0grBYr_hV8SzC31vuY-6RrkmJs0gLiAkjc76RaQqkFICFsmbwhoOsn3kcsjy3IaEV9pUd_z9KwDIGJfzFRNsPyvMqXGHWxtxx616q3WoAsk0gJbn8Y7kB-IIn-J6_nU1wGwStVMpRslycmA-djfAfsJDvpNndyqqqXpILKO8VvuSAcZ1enY6r62tIfzQV5HazxfMDIgi4iMRzvajFngq3xx6K3BmRSTC8TluuU4gY-6asJJSDsfz1JG4bsjIPCjk4CbHXtEUMklRjR29SqSAOxqiwTrMizyAzypMFhdVVZ5U-CSqI46Pv1Ol8-ayhGbwg-qEmCTHrTrVkQrIOtgdF4mxafTKzHYUgb6wo&isca=1](url)